tests/functional/monitoring/libtest/test_workload_example.py

```
@@ -64,27 +75,40 @@ def test_start_fio_job(
+    logger.info(f"Waiting for pod in namespace {project.namespace} (timeout: 300s)")
+
     try:
         ocp_pod.wait_for_resource(
             resource_count=1, condition=constants.STATUS_RUNNING, timeout=300, sleep=30
         )
+        logger.info("Pod is running successfully")
     except TimeoutExpiredError:
-        logger.error("pod for fio job wasn't deployed properly")
+        logger.error(
+            "Pod for fio job wasn't deployed properly - timeout waiting for Running state"
+        )
```

Ruff (0.15.17)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)